lib/commands/audit.js

```diff
@@ -76,54 +75,66 @@ class VerifySignatures {
     const auditedPlural = this.auditedWithKeysCount > 1 ? 's' : ''
     const timing = `audited ${this.auditedWithKeysCount} package${auditedPlural} in ` +
       `${Math.floor(Number(elapsed) / 1e9)}s`
-    this.appendOutput(`${timing}\n`)
+    this.npm.output(timing)
+    this.npm.output('')
 
     if (this.verifiedCount) {
       const verifiedBold = this.npm.chalk.bold('verified')
-      const msg = this.verifiedCount === 1 ?
-        `${this.verifiedCount} package has a ${verifiedBold} registry signature\n` :
-        `${this.verifiedCount} packages have ${verifiedBold} registry signatures\n`
-      this.appendOutput(msg)
+      if (this.verifiedCount === 1) {
+        this.npm.output(`${this.verifiedCount} package has a ${verifiedBold} registry signature`)
+      } else {
+        this.npm.output(`${this.verifiedCount} packages have ${verifiedBold} registry signatures`)
+      }
+      this.npm.output('')
```

Comment:
these seem like unrelated refactors, is it worth landing those in a separate PR to make this diff smaller?

Reply:
the commits are separate

Reply:
true enough, i just looked at the whole diff view :-) that makes it easier to land them separately tho, no?

Reply:
this branch has been very long lived (`provenance`) and we've been testing and reviewing it for months.  originally the second commit had other work in it that we pulled out for another PR but this was left here because it already existed and is just a refactor. it will be landed as two commits.